helm/k6-operator/templates/prometheus/serviceMonitor.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.prometheus.enabled }}
+# Prometheus Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
Prometheus has well known helm charts. Adding K8s manifests for these are not maintainable (unnecessary technical debt). Please replace with helm chart overriding of values.yaml if required.